### PR TITLE
ECR viewer metadata section

### LINKED ISSUE #996: Add eCR Metadata to eCR Viewer
## Action Requested
Add "eCR Metadata" section to the eCR viewer. 

## Acceptance Criteria
"eCR Metadata" section is included in all eCRs, and has the following data:
~* Reportable Conditions~
~* RCKMS Trigger Summary~
~* Jurisdictions eCR was sent to~
* eICR Identifier
* Date/Time eCR Created
* Sender Software
* Sender Facility Name
* Facility Address
* Facility Contact
* Facility ID

There should be two sub-sections created as part of this ticket: "eICR Details", and "eCR Sender Details". "RR Details" will come later.

## Testing
Unit testing is required.

## Background & Context
This section is called for in the [viewer designs](https://www.figma.com/file/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?type=design&node-id=160-441&mode=design&t=gog0c6KcUPLvQCId-0). 

## Additional Information
Some details this section needs may not be currently extracted from the RR and converted to FHIR. If so, this ticket will also include pulling that data out of the RR and converting it to FHIR.

### PR DESCRIPTION
**Pull request by gordonfarrell**
# PULL REQUEST

## Summary
Adds eCR metadata section to the viewer

## Related Issue
Fixes #996 

## Additional Information
Still a few fhirpaths I wasn't able to find, may need to pair with someone really good at digging those up. Also, is the encounter location the same as the sender or can they/will they often be different?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/ecr-viewer/src/app/api/fhir-data/fhirPath.yml [hunk 1]
```diff
@@ -25,6 +25,13 @@ patientAlcoholUse: "Bundle.entry.resource.where(resourceType='Observation').wher
 patientSexualOrientation: "idk"
 patientGenderIdentity: "Bundle.entry.resource.where(resourceType = 'Patient').extension.where(url='http: //hl7.org/fhir/us/ecr/StructureDefinition/us-ph-genderidentity-extension').extension.value.coding.display"
 
+eicrIdentifier: "Bundle.entry.resource.where(resourceType= 'Composition').id"
+dateTimeEcrCreated: "Bundle.entry.resource.where(resourceType = 'Composition').date"
+senderSoftware: "idk"
+senderFacilityName: "Bundle.entry.resource.where(resourceType = 'Encounter')[0].location[0].location.display"
+facilityAddress: "idk"
```
Comment (emmastephenson) [inline containers/ecr-viewer/src/app/api/fhir-data/fhirPath.yml:32]: Is this different from the "facilityStreetAddress" down on line 39? I think we might be able to use that instead.

File: containers/ecr-viewer/src/app/api/fhir-data/fhirPath.yml [hunk 2]
```diff
@@ -25,6 +25,13 @@ patientAlcoholUse: "Bundle.entry.resource.where(resourceType='Observation').wher
 patientSexualOrientation: "idk"
 patientGenderIdentity: "Bundle.entry.resource.where(resourceType = 'Patient').extension.where(url='http: //hl7.org/fhir/us/ecr/StructureDefinition/us-ph-genderidentity-extension').extension.value.coding.display"
 
+eicrIdentifier: "Bundle.entry.resource.where(resourceType= 'Composition').id"
+dateTimeEcrCreated: "Bundle.entry.resource.where(resourceType = 'Composition').date"
+senderSoftware: "idk"
```
Comment (emmastephenson) [inline containers/ecr-viewer/src/app/api/fhir-data/fhirPath.yml:30]: Would you be able to create a ticket to ensure we grab this information during FHIR conversion (like #1001 and #1002)? I want to make sure we follow up on all the "idk" fields.

File: node_modules/.cache/prettier/.prettier-caches/057d6815aa642772604ab7743d34d2138cd7bd3c.json
Comment (emmastephenson) [inline node_modules/.cache/prettier/.prettier-caches/057d6815aa642772604ab7743d34d2138cd7bd3c.json:1]: I don't think this file should be included?

**Review by emmastephenson — COMMENTED** [review]
Looks good, just a couple small things!

File: containers/ecr-viewer/src/app/api/fhir-data/fhirPath.yml (hunk 1 above)
Comment (gordonfarrell) [reply containers/ecr-viewer/src/app/api/fhir-data/fhirPath.yml:32]: Yeah I wasn't quite sure if encounter location and sender location would always be the same, after a chat at standup this morning I think for now at least encounter loc will be used here too

File: node_modules/.cache/prettier/.prettier-caches/057d6815aa642772604ab7743d34d2138cd7bd3c.json
Comment (gordonfarrell) [reply node_modules/.cache/prettier/.prettier-caches/057d6815aa642772604ab7743d34d2138cd7bd3c.json:1]: You're right! What the heck, don't we have node modules in our git ignore? Not sure how this slipped in

File: containers/ecr-viewer/src/app/api/fhir-data/fhirPath.yml (hunk 2 above)
Comment (gordonfarrell) [reply containers/ecr-viewer/src/app/api/fhir-data/fhirPath.yml:30]: Will do

File: node_modules/.cache/prettier/.prettier-caches/057d6815aa642772604ab7743d34d2138cd7bd3c.json
Comment (gordonfarrell) [reply node_modules/.cache/prettier/.prettier-caches/057d6815aa642772604ab7743d34d2138cd7bd3c.json:1]: Apparently we don't. will add it

**Review by KennethSkylight — APPROVED** [review]
lgtm

**Pull request merged**